test/vstest.console.UnitTests/TestPlatformHelpers/TestRequestManagerTests.cs

```diff
@@ -43,6 +44,42 @@ public void TestInit()
                 this.mockTestPlatformEventSource.Object);
         }
 
+        [TestCleanup]
+        public void Cleanup()
+        {
+            CommandLineOptions.Instance.Reset();
+        }
+
+        [TestMethod]
+        public void TestRequestManagerShouldInitializeConsoleLogger()
+        {
+            var mockLoggerEvents = new DummyLoggerEvents(TestSessionMessageLogger.Instance);
```

Comment:
Please make these instance variables.

(These are copied from earlier code, good to clean up)